### PR TITLE
[skip vbump] fix version

### PR DESCRIPTION
**Pull request by nikolas-burkoff**
Also need https://github.com/insightsengineering/scda.2022/pull/90

### CODE REVIEW

**Review by danielinteractive — APPROVED** [review]

File: staged_dependencies.yaml [hunk 1]
```diff
@@ -29,4 +29,7 @@ upstream_repos:
   insightsengineering/scda:
     repo: insightsengineering/scda
     host: https://github.com
+    insightsengineering/scda.2022:
```
Comment (nikolas-burkoff) [inline staged_dependencies.yaml:32]:
```suggestion
  insightsengineering/scda.2022:
```

**Comment by nikolas-burkoff** [comment]
@BFalquet  - this may not be needed I'm told you are working on updating scda.2021 to scda.2022 - though can that PR also have 

1) the version bump like this one to fix the NEWS (should have [skip vbump] in the title)
2) update staged.deps in tmh and scda.2022 (i.e. like https://github.com/insightsengineering/scda.2022/pull/90)

**Comment by danielinteractive** [comment]
@BFalquet 
ok so we could merge this branch into your branch e.g.
and rename the scda.2022 branch to match your branch name
https://github.com/insightsengineering/scda.2022/pull/90

**Comment by nikolas-burkoff** [comment]
ok so this is weird we have already released v0.1.4: 
image

But the news is blank:
image

So this PR should NOT be used

**Pull request closed without merging**